ci/windows/test_py3_gpu.ps1

```diff
@@ -23,7 +23,20 @@ $env:MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
 $env:MXNET_SUBGRAPH_VERBOSE=0
 $env:MXNET_HOME=[io.path]::combine($PSScriptRoot, 'mxnet_home')
 
```

Comment:
let's print all ENV Vars to compare state of env vars in CI run vs manual powershell test
```Get-ChildItem Env:```